public/app/constants/profile-metrics.json

```diff
@@ -43,7 +43,7 @@
   },
   "memory:alloc_objects:count:space:bytes": {
     "id": "memory:alloc_objects:count:space:bytes",
```

Comment:
Unrelated to the change but still relevant I think, why do we need the `:space:bytes` in the key and id here?